### PR TITLE
Feature: Parameter Node added to new Helper module.

### PR DESCRIPTION
**Pull request by jubeira**
This PR adds a new RosJava node that allows loading a set Yaml files to the Parameter Server.

Read the class documentation for more information about how to use the node.
To add the dependency to your project, add the following line to your Gradle script (0.3 is the current package version):
`compile 'org.ros.rosjava_core:rosjava_helpers:[0.3, 0.4)'`

### CODE REVIEW

File: rosjava_helpers/src/main/java/org/ros/helpers/ParameterLoaderNode.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import org.ros.node.parameter.ParameterTree;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Loads parameters from Yaml files into ROS Parameter Server.
```
Comment (adamantivm) [inline rosjava_helpers/src/main/java/org/ros/helpers/ParameterLoaderNode.java]: wording nit: 'into the ROS parameter server'

File: rosjava_helpers/src/main/java/org/ros/helpers/ParameterLoaderNode.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            ParameterTree parameterTree = connectedNode.getParameterTree();
+            log = connectedNode.getLog();
+
+            // TODO: For some reason, setting the / param when using a rosjava master doesn't work
+            // It does work fine with an external master, and also setting other params of any type
+            for (LoadedResource r : params) {
+                addParams(parameterTree, r.namespace, r.resource);
+            }
+
+            connectedNode.shutdown();
```
Comment (adamantivm) [inline rosjava_helpers/src/main/java/org/ros/helpers/ParameterLoaderNode.java:107]: it doesn't need to wait here a little bit before shutdown? This doesn't cause any of the configurations to be lost?

**Review by adamantivm — APPROVED** [review]
awesome contribution @jubeira ! I added two very minor comments.

File: rosjava_helpers/src/main/java/org/ros/helpers/ParameterLoaderNode.java (hunk 2 above)
Comment (jubeira) [reply rosjava_helpers/src/main/java/org/ros/helpers/ParameterLoaderNode.java:107]: I haven't had any problem at all with this. I tested it by loading different sets of files multiple times and all the parameters were uploaded correctly each time.

**Comment by jubeira** [comment]
Nit addressed!

**Pull request merged**